lib/core/src/lib/form/components/mock/form-renderer.component.mock.ts

```diff
@@ -1505,7 +1505,7 @@ export const formDateVisibility = {
                     existingColspan: 1,
                     maxColspan: 2
                   },
-                  dateDisplayFormat: 'YYYY-MM-DD'
+                  dateDisplayFormat: 'yyyy-M-d'
```

Comment:
I think this should be `yyyy-MM-dd` to provide compatibility for previous implementation

Reply:
changed to `yyyy-MM-dd `